.github/workflows/link-check-cron.yml

```
@@ -21,12 +21,18 @@ jobs:
       with:
         use-quiet-mode: 'yes'
         use-verbose-mode: 'yes'
+        
+        # A configuration file can be included, indicating the properties of the link check action
+        # More information can be found here: https://github.com/tcort/markdown-link-check#config-file-format
+        # The default file is mlc_config.json, located at the root of the directory
+        
+        # config-file: name-of-config-file.json
```

```suggestion
        # A configuration file can be included, indicating the properties of the link check action
        # More information can be found here: https://github.com/tcort/markdown-link-check#config-file-format
        # Create mlc_config.json file in the root of the directory
```

please update also the other comment in other workflows.
we should not encourage people to provide custom names for config files because then they would need to modify workflows, but they should not modify workflows as we will then overwrite their changes we make updates here in `.github` repo